flink-runtime/src/main/java/org/apache/flink/runtime/jobmaster/factories/DefaultJobMasterServiceFactory.java

```diff
@@ -122,6 +123,7 @@ private JobMasterService internalCreateJobMasterService(
                         DefaultExecutionDeploymentReconciler::new,
                         BlocklistUtils.loadBlocklistHandlerFactory(
                                 jobMasterConfiguration.getConfiguration()),
+                        Collections.emptySet(),
```

Comment:
Are we missing something here?

Reply:
Right, this is a placeholder for loaded FailureEnrichers using a conf (empty for now to facilitate testing) -- tracking this as part of FLINK-31993